packages/loader/container-loader/src/container.ts

```diff
@@ -841,8 +842,26 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             this._resolvedUrl = resolvedUrl;
             await this.connectStorageService();
 
-            // This we can probably just pass the storage service to the blob manager - although ideally
-            // there just isn't a blob manager
+            // upload blobs here (NYI)
+
+            // post summary here
+            if (this.loader.services.detachedBlobStorage && this.loader.services.detachedBlobStorage.size > 0) {
```

Comment:
You are using different styles (consider this.loader.services.detachedBlobStorage?.size !== 0 above).
I'd suggest to defined new boolean variable uptop separating these two flows and using it across this function - it's easier to follow code that way.